typescript/sdk/src/token/EvmERC20WarpModule.hardhat-test.ts

```
@@ -925,7 +932,7 @@
         expect(txs.length).to.equal(0);
       });
 
-      it(`should add the specified addresses as rebalancing bridges for tokens of type "${tokenType}"`, async () => {
+      it.only(`should add the specified addresses as rebalancing bridges for tokens of type "${tokenType}"`, async () => {
```

Biome (1.9.4)

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)